### PR TITLE
[AGENT-5469] Bump version to 1.11.0

### PR DESCRIPTION
**Pull request by elatt**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Bump version to `v1.11.0` and update changelog

## Rationale
Bump the minor version because this change represents new, backward compatible changes.

### CODE REVIEW

**Comment by elatt** [comment]
This needs to merge **after** #997

**Review by rvorobii — APPROVED** [review]

**Review by scottp-ml — APPROVED** [review]

File: custom_model_runner/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+#### [1.11.0] - 2023-06-30
```
Comment (yakov-g) [inline custom_model_runner/CHANGELOG.md:7]:
```suggestion
#### [1.11.0] - 2024-05-01
```

**Review by dulcardo — COMMENTED** [review]
Are you going to release new `public_dropin_environments`?
if so you also need to:
* update drum version. 
* update env version ids. To do this, run ./tools/env_version_update.py

https://github.com/datarobot/datarobot-user-models/pull/949/files

File: custom_model_runner/CHANGELOG.md (hunk 1 above)
Comment (elatt) [reply custom_model_runner/CHANGELOG.md]: I also noticed there are git-tags for versions. Do I need to apply that manually or will it get added automatically when I merge this PR?

**Pull request merged**